tags/Coveo/CoveoAnalyticsLogEvent/Coveo Analytics Log Event.tpl

```diff
@@ -0,0 +1,446 @@
+    "displayName": "",
+    "id": "brand_dummy"
+  },
+  "containerContexts": [
+    "WEB"
+  ]
+}
+
+
+___TEMPLATE_PARAMETERS___
+
+[
+  {
+    "macrosInSelect": false,
+    "selectItems": [
+      {
+        "displayValue": "Custom",
+        "value": "custom"
+      },
+      {
+        "displayValue": "View",
+        "value": "view"
+      }
+    ],
+    "displayName": "Event Type",
+    "simpleValueType": true,
+    "name": "eventType",
+    "type": "SELECT"
+  },
+  {
+    "enablingConditions": [
+      {
+        "paramName": "eventType",
+        "type": "EQUALS",
+        "paramValue": "view"
+      }
+    ],
+    "displayName": "View Event Metadata",
+    "name": "View Event Metadata",
+    "groupStyle": "ZIPPY_OPEN",
+    "type": "GROUP",
+    "subParams": [
+      {
+        "help": "The name of the field used to trace back the current page in the Coveo Index.",
+        "valueValidators": [
+          {
+            "type": "NON_EMPTY"
+          }
+        ],
+        "displayName": "Content ID Key",
+        "defaultValue": "permanentid",
+        "simpleValueType": true,
+        "name": "contentIdKey",
+        "type": "TEXT"
+      },
+      {
+        "help": "The value of the field used to trace back the current page in the Coveo Index.",
+        "valueValidators": [
+          {
+            "type": "NON_EMPTY"
+          }
+        ],
+        "displayName": "Content ID Value",
+        "simpleValueType": true,
+        "name": "contentIdValue",
+        "type": "TEXT"
+      }
+    ]
+  },
+  {
+    "enablingConditions": [
+      {
+        "paramName": "eventType",
+        "type": "EQUALS",
+        "paramValue": "custom"
+      }
+    ],
+    "displayName": "Custom Event Metadata",
+    "name": "Custom Event Metadata",
+    "groupStyle": "ZIPPY_OPEN",
+    "type": "GROUP",
+    "subParams": [
+      {
+        "help": "",
+        "valueValidators": [
+          {
+            "type": "NON_EMPTY"
+          }
+        ],
+        "displayName": "Event Type",
+        "simpleValueType": true,
+        "name": "customEventType",
+        "type": "TEXT"
+      },
+      {
+        "help": "",
+        "valueValidators": [
+          {
+            "type": "NON_EMPTY"
+          }
+        ],
+        "displayName": "Event Value",
+        "simpleValueType": true,
+        "name": "customEventValue",
+        "type": "TEXT"
+      }
+    ]
+  },
+  {
+    "displayName": "Document Metadata",
+    "name": "Document Metadata",
+    "groupStyle": "ZIPPY_OPEN",
+    "type": "GROUP",
+    "subParams": [
+      {
+        "help": "The current language of the page's content. If not properly detected, will fallback on \"en\".",
```

Comment:
A bit misleading since it doesn't actually fall back on "en" :)

You could:

1. Change the help text so there is no fallback
2. Remove the validation rule and have the `|| "en"` in the code

Setting "en" as the Default Value of the field wouldn't work, since the user could just delete that and add something else or leave the field empty.